safe/gui/ui/multi_exposure_dialog_base.ui

```diff
@@ -33,7 +33,7 @@
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
-          <string>This tool allows you to run an multi-exposure analysis. By selecting one or many exposures, the impact function will aggregate the results in one report.</string>
+          <string>This tool allows you to run a multi-exposure analysis. Select a hazard, one or more exposures and an aggregation layer. The impact function will aggregate the results of all the hazard / exposure combinations in one report. Use the reporting options tab to specify which layers should be included and visitble in the report. You can set a template on the reporting options tab for producing a printed report. The status tab will show the progress of the analysis as it runs.</string>
```

Comment:
* One exposure only is not allowed, 2 minimum. Should it be?
* s/visitble/visible/g

Reply:
Ok thanks - updated now:

![screen shot 2017-11-08 at 15 43 37](https://user-images.githubusercontent.com/178003/32552196-c595d8e6-c49b-11e7-9164-bd50b885d5ed.png)